master/buildbot/status/builder.py

```diff
@@ -62,23 +62,23 @@ class BuilderStatus(styles.Versioned):
     I live in the buildbot.process.build.Builder object, in the
     .builder_status attribute.
 
-    @type  category: string
-    @ivar  category: user-defined category this builder belongs to; can be
+    @type  tags: None or list of strings
+    @ivar  tags: user-defined "tag" this builder has; can be
                      used to filter on in status clients
     """
 
     implements(interfaces.IBuilderStatus, interfaces.IEventSource)
 
-    persistenceVersion = 1
+    persistenceVersion = 2
```

Comment:
Note that this persistence is identical to the version done in Nine, so there is no version conflict (ie, both versions have a "version 2" and they are identical).